attributes/default.rb

```diff
@@ -26,7 +26,10 @@
 default['osl-openstack']['cinder']['iscsi_ips'] = []
 default['osl-openstack']['image']['glance_vol'] = nil
 default['osl-openstack']['endpoint_hostname'] = nil
+default['osl-openstack']['network_hostname'] = nil
 default['osl-openstack']['db_hostname'] = nil
+default['osl-openstack']['bind_service'] = node['ipaddress']
+default['osl-openstack']['network_node'] = false
```

Comment:
Like I said above, perhaps it might be better to rename this attribute to something else. It's intended to ensure that the primary controller machine doesn't include it since it will be on a separate node by itself. We may end up doing this for other services eventually too so it might make sense to refactor how this is done.